library/types/src/modules/Hostname.rb

```diff
@@ -155,30 +155,20 @@ def MergeFQ(hostname, domain)
     # (uses hostname --fqdn)
     # @return FQ hostname
     def CurrentFQ
-      fqhostname = ""
+      hostname_data = SCR.Execute(path(".target.bash_output"), "hostname --fqdn")
 
-      hostname_data = Convert.to_map(
-        SCR.Execute(path(".target.bash_output"), "hostname --fqdn")
-      )
       if hostname_data.nil? || hostname_data["exit"] != 0 || invalid_hostname?(hostname_data["stdout"].to_s.strip)
-        fqhostname = if FileUtils.Exists("/etc/hostname")
-                       SCR.Read(path(".target.string"), "/etc/hostname")
-                     else
-                       ""
-                     end
-
-        if fqhostname == "" || fqhostname.nil?
-          # last resort (#429792)
-          fqhostname = "linux.#{@DefaultDomain}"
-        end
-        Builtins.y2warning("Using fallback hostname %1", fqhostname)
+        Builtins.y2warning("Using fallback hostname")
```

Comment:
well I usually agree / do so. But in this case the output can be easily extracted from other output and I don't want to fill the log more than needed.